recipe/meta.yaml

```diff
@@ -10,27 +10,31 @@ source:
   sha256: 4ffa3bdea2b826cd61da66cf8a71f393b83d1229ec3943e3a1109c5e6cb3d6a3
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - flashtext >=2.7,<3.0
-    - loguru >=0.5.3,<0.7.0
+    - loguru >=0.5.3
     - pip
     - poetry-core >=1.0.0,<2.0.0
-    - pydantic >=1.8,<2.0
+    - pydantic >=2.7.1,<3.0
     - python >=3.10
   run:
     - flashtext >=2.7,<3.0
-    - loguru >=0.5.3,<0.7.0
-    - pydantic >=1.8,<2.0
+    - loguru >=0.5.3
+    - pydantic >=2.7.1,<3.0
     - python >=3.10
 
 test:
   imports:
     - flashgeotext
+  requires:
+    - pip
+  commands:
+    - pip check
 
 about:
   home: https://flashgeotext.iwpnd.pw
```

Comment:
can you change that to the GitHub readme while you're at it?

Reply:
The `about`?

Reply:
yeah, those links to iwpnd.pw are no longer valid. Could just link to readme instead.